overlay.nix

```diff
@@ -310,6 +310,10 @@ let
 
         amd-scalapack = callPackage ./pkgs/lib/amd-scalapack { };
 
+        cppe = callPackage ./pkgs/lib/cppe { };
```

Comment:
We already have CPP upstream (`<nixpkgs>/pkgs/development/libraries/science/chemistry/cppe/default.nix`)